docs/admin/index.md

```diff
@@ -1,11 +1,283 @@
+|-----------------------|-----------------------------------------------------------|
+| `-D`, `--debug=false` | Enable or disable debug mode. By default, this is false. |
+| `-H`,`--host=[]`      | Daemon socket(s) to connect to.                           |
+| `--tls=false`         | Enable or disable TLS. By default, this is false.         |
+
+
+Here is an example of running the `docker` daemon with configuration options:
+
+    $ dockerd -D --tls=true --tlscert=/var/docker/server.pem --tlskey=/var/docker/serverkey.pem -H tcp://192.168.59.3:2376
+
+These options :
+
+- Enable `-D` (debug) mode
+- Set `tls` to true with the server certificate and key specified using `--tlscert` and `--tlskey` respectively
+- Listen for connections on `tcp://192.168.59.3:2376`
+
+The command line reference has the [complete list of daemon flags](../reference/commandline/dockerd.md)
+with explanations.
+
+### Daemon debugging
+
+As noted above, setting the log level of the daemon to "debug" or enabling debug mode
+with `-D` allows the administrator or operator to gain much more knowledge about the
+runtime activity of the daemon. If faced with a non-responsive daemon, the administrator
+can force a full stack trace of all threads to be added to the daemon log by sending the
+`SIGUSR1` signal to the Docker daemon. A common way to send this signal is using the `kill`
+command on Linux systems. For example, `kill -USR1 <daemon-pid>` sends the `SIGUSR1`
+signal to the daemon process, causing the stack dump to be added to the daemon log.
+
+> **Note:** The log level setting of the daemon must be at least "info" level and above for
+> the stack trace to be saved to the logfile. By default the daemon's log level is set to
+> "info".
+
+The daemon will continue operating after handling the `SIGUSR1` signal and dumping the stack
+traces to the log. The stack traces can be used to determine the state of all goroutines and
+threads within the daemon.
+
+## Ubuntu
+
+As of `14.04`, Ubuntu uses Upstart as a process manager. By default, Upstart jobs
+are located in  `/etc/init` and the `docker` Upstart job can be found at `/etc/init/docker.conf`.
+
+After successfully [installing Docker for Ubuntu](../installation/linux/ubuntulinux.md),
+you can check the running status using Upstart in this way:
+
+    $ sudo status docker
+
+    docker start/running, process 989
+
+### Running Docker
+
+You can start/stop/restart the `docker` daemon using
+
+    $ sudo start docker
+
+    $ sudo stop docker
+
+    $ sudo restart docker
+
+
+### Configuring Docker
+
+The instructions below depict configuring Docker on a system that uses `upstart`
+as the process manager. As of Ubuntu 15.04, Ubuntu uses `systemd` as its process
+manager. For Ubuntu 15.04 and higher, refer to [control and configure Docker with systemd](systemd.md).
+
+You configure the `docker` daemon in the `/etc/default/docker` file on your
```

Comment:
We should get rid of most of this anyway, and use `daemon.json` as the primary/preferred way to configure the daemon